src/storage/txn/commands/prewrite.rs

```diff
@@ -587,22 +591,25 @@ impl PrewriteKind for Optimistic {
     }
 }
 
+/// Pessimistic `PreWriteKind`.
 struct Pessimistic {
     for_update_ts: TimeStamp,
 }
 
 impl PrewriteKind for Pessimistic {
+    // see the `MutationLock` trait implementation for `(Mutation, bool)`
     type Mutation = (Mutation, bool);
 
     fn txn_kind(&self) -> TransactionKind {
         TransactionKind::Pessimistic(self.for_update_ts)
     }
 }
 
-/// See PrewriteKind::Mutation. This is either just a mutation (for optimistic
-/// transactions, since they never include locks) or a mutation and a bool (for
-/// pessimistic transactions, where the bool indicates if the mutation is taking
-/// a pessimistic lock).
+/// A placeholder type to indicate whether a `PrewriteKind` is part of an optimistic
+/// or a pessimistic transaction.
```

Comment:
```suggestion
/// The type of mutation and, optionally, its extra information, differing for the
/// optimistic and pessimistic transaction.
///
```

Reply:
Done. I think I missed this in my previous edits.